### ISSUE TITLE
Current DEV version does not open a backed-up file on mouse double-click

### ISSUE THREAD

**Issue by aryoda** [bit-team/backintime#1500]
## Steps to reproduce

- Choose a profile with existing snapshots
- Select one snapshot with existing *.txt files
- Select one entry of your `Backup folders`
- on the right side navigate to a sub folder with an existing *.txt file
- Left-double-click on a *.txt file on the right side

The file should be opened in the associated app but nothing happens...

## Symptoms

This traceback appears in the console:

```
Traceback (most recent call last):
  File "/home/juergen/dev/backintime/qt/app.py", line 1505, in filesViewItemActivated
    self.openPath(rel_path)
  File "/home/juergen/dev/backintime/qt/app.py", line 1550, in openPath
    self.run = QDesktopServices.openUrl(QUrl('file://' + full_path))
NameError: name 'QUrl' is not defined
```

Even after adding the missing `QUrl` import nothing happens on a double-click...

## Last known good

With an older dev version (installed July 15, 2023) opening a file with a double click does work.

**Comment by buhtz** [comment]
Sounds like I broke something here. I'll take a look at it.

**Comment by aryoda** [comment]
No urgent fix required I'd say.

I even had a FR for this because this functionality is totally hidden behind a mouse-double-click:

- Add a context menu entry "Open a file copy"

Otherwise
- nobody sees this feature
- nobody understands that a tmp copy of the file is opened (not the backed-up file directly to avoid changing it)

**Comment by buhtz** [comment]
I fixed the missing import of QUrl with PR #1501. Works fine by me.

Agree to your idea about the context menu. I investigated the code and realized that is bit of work and it might be easier while going further with refactor the code in app.py especially the Files View part.

Before opening the context menu it need to be known 1) if there is a file selected and 2) if it is a real file ("Now") or part of a snapshot. If 1) is true there need to be such a context menu entry. If 2) is true it should be labeled "Open file... [Double-click]" and if false it should be labled "Open a file copy... [Double-click]".

I suggest to open a seperate FR for it and close this issue as a bug report.

**Comment by aryoda** [comment]
> I suggest to open a seperate FR for it and close this issue as a bug report.

I fully agree, don't mix two things in one issue ;-)

Anyhow I have discovered some more inconsistencies in the GUI, I think I will open a new "improve GUI" issue soon and collect everything (incl. the idea of context menu entry for opening a file) in that issue then...

THX for your quick fix!
